src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/BalanceCalculatorIntegrationTest.java

```diff
@@ -43,27 +45,57 @@ void calculate_endToEndInGmtToBst_contributionsAndCumulativeTotalsAsExpected() {
 
     List<Accrual> accruals = balanceCalculator.calculate(timeEntry);
 
-    assertThat(accruals).hasSize(4);
+    assertThat(accruals).hasSize(8);
 
+    // Annual Target Hours
+    assertThat(accruals.get(0).getAccrualType()).isEqualTo(ANNUAL_TARGET_HOURS);
     assertThat(accruals.get(0).getContributions().getTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(600));
     assertThat(accruals.get(0).getCumulativeTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(6600));
 
+    assertThat(accruals.get(1).getAccrualType()).isEqualTo(ANNUAL_TARGET_HOURS);
     assertThat(accruals.get(1).getContributions().getTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(600));
     assertThat(accruals.get(1).getCumulativeTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(7200));
 
+    assertThat(accruals.get(2).getAccrualType()).isEqualTo(ANNUAL_TARGET_HOURS);
     assertThat(accruals.get(2).getContributions().getTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(240));
     assertThat(accruals.get(2).getCumulativeTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(7440));
 
+    assertThat(accruals.get(3).getAccrualType()).isEqualTo(ANNUAL_TARGET_HOURS);
     assertThat(accruals.get(3).getContributions().getTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(720));
     assertThat(accruals.get(3).getCumulativeTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(8160));
+
```

Comment:
Could we make this a method i.e.
```
assertTypeAndTotals(accrual, expectedAccrualType, expectedContributionTotal, expectedCumulativeTotal)
```

Reply:
Updated tests to use `assertTypeAndTotals` method